ttnn/cpp/ttnn/operations/eltwise/unary/common/unary_op_types.hpp

```diff
@@ -102,3 +102,15 @@ struct UnaryWithParam {
 using FusedActivations = std::vector<ttnn::operations::unary::UnaryWithParam>;
 
 }
+
+namespace tt::stl::json {
+
+template <>
+struct from_json_t<ttnn::operations::unary::UnaryWithParam> {
```

Comment:
Not sure if this was already looked at. Have you tried seeing if this can be removed and what's in tt_metal/tt_stl/reflection.hpp used instead?